README.md

```diff
@@ -3,7 +3,8 @@
 [![Build Status](https://travis-ci.org/rwengine/openrw.svg?branch=master)](https://travis-ci.org/rwengine/openrw)
 
 OpenRW is an open source re-implementation of Rockstar Games' Grand Theft Auto III,
-a classic 3D action game first published in 2001.
+a classic 3D action game first published in 2001. It is designed to run on Linux,
+macOS and Windows systems, along with several variants of BSD.
```

Comment:
"OpenRW is a cross-platform, open-source re-implementation [..]" and then
"Our implementation has been ported to Linux, macOS, Windows and several variants of BSD." ?

- imo your proposal sounds like we don't want to support any other systems because it's only designed for these.
I also felt that yours sounded as if "Grand Theft Auto III [..] was designed to run on Linux, macOS, [..]"